.gitignore

```diff
@@ -44,3 +44,6 @@
 
 # For Pycharm
 /.idea
+
+# For backup files.
+*~
```

Comment:
I added `~` here as our `uuid-replace-*` scripts created backup files and we do not want to push it to the repo.